test/test.js

```diff
@@ -14,7 +14,13 @@ function createRepo( callback ) {
 
 		testRepo.exec( "commit", "--allow-empty",
 			"--author=John Doe <john.doe@example.com>", "-m first commit",
-			callback );
+			function( error ) {
+				if ( error ) {
+					throw error;
```

Comment:
> It's what you did before the commit so I figured it was the right thing to do.

Hmm...indeed. I have no idea why I made this function throw errors, especially asynchronously. I guess this is fine for now. I'll clean up the method later.